taxcalc/records.py

```diff
@@ -396,7 +396,7 @@ def increment_year(self):
         # Implement Stage 1 Extrapolation blowup factors
         self._blowup(self._current_year)
         # Implement Stage 2 Extrapolation reweighting.
-        self.s006 = self.WT["WT" + str(self.current_year)] / 100
+        self.s006 = (self.WT["WT" + str(self.current_year)] / 100).as_matrix()
```

Comment:
I think the call you want is `values`. I just ran this and it seems we have `WT` is a DataFrame, so accessing one of its columns, e.g. `WT['WT2016']` produces a Series, and to get a numpy array from that series you can access it via the `values` attribute.